include/zephyr/usb/class/usb_audio.h

```diff
@@ -100,63 +100,105 @@ enum usb_audio_fucs {
  * Refer to Table 2-1 - Table 2-4 from termt10.pdf
  */
 enum usb_audio_terminal_types {
-	/* USB Terminal Types */
+	/**
+	 * @name USB Terminal Types
+	 * @{
+	 */
+	/** USB undefined */
 	USB_AUDIO_USB_UNDEFINED   = 0x0100,
+	/** USB streaming */
 	USB_AUDIO_USB_STREAMING   = 0x0101,
+	/** USB vendor specific */
 	USB_AUDIO_USB_VENDOR_SPEC = 0x01FF,
+	/** @} */
 
-	/* Input Terminal Types */
+	/**
+	 * @name Input Terminal Types
+	 * @{
+	 */
+	/** Input undefined */
 	USB_AUDIO_IN_UNDEFINED      = 0x0200,
+	/** Microphone */
 	USB_AUDIO_IN_MICROPHONE     = 0x0201,
+	/** Desktop microphone */
 	USB_AUDIO_IN_DESKTOP_MIC    = 0x0202,
+	/** Personal microphone */
 	USB_AUDIO_IN_PERSONAL_MIC   = 0x0203,
+	/** Omni directional microphone */
 	USB_AUDIO_IN_OM_DIR_MIC     = 0x0204,
+	/** Microphone array */
 	USB_AUDIO_IN_MIC_ARRAY      = 0x0205,
+	/** Processing microphone array */
 	USB_AUDIO_IN_PROC_MIC_ARRAY = 0x0205,
+	/** @} */
 
-	/* Output Terminal Types */
+	/**
+	 * @name Output Terminal Types
+	 * @{
+	 */
+	/** Output undefined */
 	USB_AUDIO_OUT_UNDEFINED		= 0x0300,
+	/** Speaker */
 	USB_AUDIO_OUT_SPEAKER		= 0x0301,
+	/** Headphones */
 	USB_AUDIO_OUT_HEADPHONES	= 0x0302,
+	/** Head mounted display audio */
 	USB_AUDIO_OUT_HEAD_AUDIO	= 0x0303,
+	/** Desktop speaker */
 	USB_AUDIO_OUT_DESKTOP_SPEAKER	= 0x0304,
+	/** Room speaker */
 	USB_AUDIO_OUT_ROOM_SPEAKER	= 0x0305,
+	/** Communication speaker */
 	USB_AUDIO_OUT_COMM_SPEAKER	= 0x0306,
+	/** Low frequency effects speaker */
 	USB_AUDIO_OUT_LOW_FREQ_SPEAKER	= 0x0307,
+	/** @} */
 
-	/* Bi-directional Terminal Types */
+	/**
+	 * @name Bi-directional Terminal Types
+	 * @{
+	 */
+	/** Bidirectional undefined */
 	USB_AUDIO_IO_UNDEFINED			= 0x0400,
+	/** Handset */
 	USB_AUDIO_IO_HANDSET			= 0x0401,
+	/** Headset */
 	USB_AUDIO_IO_HEADSET			= 0x0402,
+	/** Speakerphone, no echo reduction */
 	USB_AUDIO_IO_SPEAKERPHONE_ECHO_NONE	= 0x0403,
+	/** Speakerphone, echo reduction */
 	USB_AUDIO_IO_SPEAKERPHONE_ECHO_SUP	= 0x0404,
+	/** Speakerphone, echo cancellation */
 	USB_AUDIO_IO_SPEAKERPHONE_ECHO_CAN	= 0x0405,
 };
 
+/**
+ * @brief Audio device direction.
+ */
 enum usb_audio_direction {
 	USB_AUDIO_IN = 0x00,
 	USB_AUDIO_OUT = 0x01
 };
 
 /**
  * @brief Feature Unit event structure.
- *
- * The event structure is used by feature_update_cb in order to inform the App
- * whenever the Host has modified one of the device features.
```

Comment:
Why are you removing this text?

Reply:
good catch, thanks. Completely unintentional and happened as part of cutting/pasting the @params following this text. Will add it back of course

Reply:
added it back, thanks for paying attention :)